sympy/concrete/tests/test_sums_products.py

```diff
@@ -1131,3 +1157,90 @@ def test_exceptions():
     raises(ReorderError, lambda: S.reorder_limit(0, 1))
     S = Sum(x*y, (x, a, b), (y, 1, 4))
     raises(NotImplementedError, lambda: S.is_convergent())
+
+
+def test_issue_17165():
+    n = symbols("n", integer=True)
+    x = symbols('x')
+    s = (x*Sum(x**n, (n, -1, oo)))
+    ssimp = s.doit().simplify()
+
+    assert ssimp == Piecewise((-1/(x - 1), Abs(x) < 1),
+                              (x*Sum(x**n, (n, -1, oo)), True))
+    assert ssimp == ssimp.simplify()
+
+
+def test__dummy_with_inherited_properties_concrete():
+    x = Symbol('x')
+
+    from sympy import Tuple
+    d = _dummy_with_inherited_properties_concrete(Tuple(x, 0, 5))
+    assert d.is_real
+    assert d.is_integer
+    assert d.is_nonnegative
+    assert d.is_extended_nonnegative
```

Comment:
Do you think it should inherit this even though nothing to force extended is present in the limit? i.e. if there is no infinite value should the dummy have the extended enhancement?

Reply:
Good question. I'm not sure it inherits it actually.

```
In [11]: x = Symbol('x', nonnegative=True)

In [12]: x.is_extended_nonnegative
Out[12]: True
```